sql-statements/sql-statement-create-user.md

```diff
@@ -73,6 +75,30 @@ CREATE USER 'newuser5'@'%' ACCOUNT LOCK;
 Query OK, 1 row affected (0.02 sec)
 ```
 
+Create a user with a comment.
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE USER 'newuser6'@'%' COMMENT 'This user is created only for test';
+```
+
+```
+Query OK, 1 row affected (0.02 sec)
+```
+
+Create a user with an `email` attribute.
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE USER 'newuser7'@'%' ATTRIBUTE '{"email": "user@pingcap.com"}';
+```
+
+```
+Query OK, 1 row affected (0.02 sec)
+```
+
```

Comment:
````suggestion
Create a user with a comment.

```sql
CREATE USER 'newuser6'@'%' COMMENT 'This user is created only for test';
```

```
Query OK, 1 row affected (0.02 sec)
```

Create a user with an `email` attribute.

```sql
CREATE USER 'newuser7'@'%' ATTRIBUTE '{"email": "user@pingcap.com"}';
```

```
Query OK, 1 row affected (0.02 sec)
```

````